models/banner.model.js

```diff
@@ -0,0 +1,17 @@
+const { Schema, model } = require("mongoose");
+
+const bannerSchema = new Schema(
+  {
+    title: { type: String, required: true },
```

Comment:
You can add message to Schema like
For example
title: { type: String, required: [true, "title required for banner"]
So when banner don't get required details you will see error message with which details are missing.